WorkflowCombine/Testing/WorkerTesting.swift

```diff
@@ -15,6 +15,7 @@
  */
 
 #if DEBUG
+import IssueReporting
```

Comment:
for my education: where is this lib coming from? Apple side?

Reply:
this is a pointfree library – it's a dependency of the CustomDump lib that we already use: https://github.com/pointfreeco/swift-issue-reporting

Reply:
though now that you bring it up... i wonder if it should be listed as an explicit dependency in the package?

Reply:
yup and i see where the magic is coming from: https://github.com/pointfreeco/swift-issue-reporting/blob/main/Sources/IssueReporting/ReportIssue.swift#L71